packages/braintree-integration/src/validation-schemas/getBraintreeTrustlyValidationSchema.ts

```diff
@@ -0,0 +1,30 @@
+import { FormField, LanguageService } from '@bigcommerce/checkout-sdk';
+import { memoize } from '@bigcommerce/memoize';
+import { object, string, StringSchema } from 'yup';
+
+export default memoize(function getBraintreeTrustlyValidationSchema({
+    formFieldData,
+    language,
+}: {
+    formFieldData: FormField[];
+    language: LanguageService;
+}) {
+    return object(
+        formFieldData.reduce((schema, { id, required }) => {
+            if (required) {
+                schema[id] = string()
+                    .nullable()
+                    .required(language.translate(`payment.errors.phone_number_required_error`));
+
+                if (id === 'phoneNumber') {
+                    schema[id] = schema[id].matches(
+                        /^\d{7,11}$/,
```

Comment:
Why do 7 to 11 digits range for phone number?

Reply:
This validation has been taken from Rate Pay validation scheme. But i need to rethink it